### DISCUSSION TITLE
游戏画面中的数字

### DISCUSSION THREAD

**Discussion by p1aintiff** [PaddlePaddle/PaddleOCR#13591] [Q&A]
图片剪切后坐了二值化处理，图片如下。在python，cpu环境下，使用paddleocr的简中，英语。参数上，在初始化指定psm 7，字典0-9。
第一次大都能识别，后续识别出 其它符号
![Screenshot_2024-08-05-10-20-33-267_com tencent mm](https://github.com/user-attachments/assets/458fa8e6-974a-4697-8093-0969ae9cc478)

**Comment by GreatV** [comment]
描述得太抽象了，麻烦提供一个最小可复现脚本
